.gitlab/ornl-ci-conda.yml

```diff
@@ -0,0 +1,130 @@
+variables:
+  CONDA_REQUIREMENTS: .gitlab/conda-requirements.yml
+
+
+stages:
+  - setup
+  - build
+  - test
+  - cleanup
+
+
+### MACHINES
+.on_ascent:
+  variables:
+    SITE: "ascent"
+    VENV_ENV_NAME: "ascent-env"
+    CONDA_WORK_DIR: ${SDK_WORK_DIR_OPEN}/conda
+    PYTHON_MODULE: "python/3.8-anaconda3"
+    CONDA_ARCH: "ppc64le" 
+
+# .on_summit:
+#   variables:
+#     SITE: "summit"
+#     VENV_ENV_NAME: "summit-env"
+#     CONDA_WORK_DIR: ${SDK_WORK_DIR_ALPINE}/conda
+#     PYTHON_MODULE: "python/3.8-anaconda3"
+#     CONDA_ARCH: "x86_64"
```

Comment:
`CONDA_ARCH: "ppc64le"` (just to be consistent, even it is not used now)

Reply:
Done